samples/tfm/tfm_psa_template/sample.yaml

```diff
@@ -11,3 +11,5 @@ tests:
   sample.tfm.psa_template:
     tags: tfm ci_build
     build_only: true
+    extra_configs:
+      - CONFIG_LOG_MODE_IMMEDIATE=y
```

Comment:
This option can affect race conditions, if we put this in the prj.conf then we are more likely to get the same result in CI and when building with west locally.

I can't see any reason to not put it in prj.conf. The performance of the non-secure app doesn't matter for this sample.